### ISSUE TITLE
Journal Abbrevations: custom user list is not reloaded after saving changes

### ISSUE THREAD

**Issue by jmmelko** [JabRef/jabref#7774]
Hi,

JabRef 5.2--2020-12-24--6a2a512
Windows 10 10.0 amd64 
Java 14.0.2

1/ Create a new journal abbrevation list, and create a new entry, or load an existing one, with an abbreviation *not* in the Jabref builtin list
2/ click on "save changes"
3/ try to toggle the journal name => does not work
4/ quit and relaunch: it works

Suggestion: reload the list after modification

**Comment by Siedlerchr** [comment]
Please always try the latest main version before reporting an issue https://builds.jabref.org/main/
There have been already some fixes regarding journal abbreviations, so your issue might be fixed already

**Comment by jmmelko** [comment]
Yeah sorry, I did a search in Issues, and found nothing, so I assumed it was not fixed.
But sometimes issues are fixed that aren’t reported.

i can’t update easily since my pc is IT managed.

**Comment by jmmelko** [comment]
Bug is still there in most recent portable version:
JabRef 5.2--2020-12-24--6a2a512
Windows 10 10.0 amd64 
Java 14.0.2

**Comment by jmmelko** [comment]
stil there in latest:

JabRef 5.3--2021-05-31--edb8df5
Windows 10 10.0 amd64 
Java 16.0.1 
JavaFX 16+8


(btw, why is the official jabref page pointing to fosshub which hosts outdated builds instead of builds.jabref.org ?)

**Comment by koppor** [comment]
> (btw, why is the official jabref page pointing to fosshub which hosts outdated builds instead of builds.jabref.org ?)

Becuase FOSSHub hosts the releases - and builds.jabref.org hosts the developer builds. There indeed is  difference between a released version and a build on each commit. We also call for beta feedback: https://discourse.jabref.org/c/beta-testing/11

We cannot do weekly releases after a bug fix, because releases still contain some manual steps. See https://github.com/JabRef/jabref/wiki/Releasing-a-new-Version#how-to-release-a-new-version for our steps just do get an impression.

**Comment by wujastyk** [comment]
JabRef 6.0-alpha2.587--2025-08-16--88c3e94
Linux 6.14.0-27-generic amd64 
Java 24.0.2 
JavaFX 24.0.2+1

Issue still present.

As far as I can see, adding a journal abbreviation to one's personal list using the Preferences/journal abbreviations menu doesn't work.  One has to edit one's external file manually.
